html/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Registration</title>
+        <link rel="icon" href="../img/download.jpg" />
+    </head>
+    <body>
+        <form action="#" method="post">
+            <h2>Registration Form</h2>
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="name" ></input>
```

Comment:
Input element is a self-closing tags like img element. Remove the closing tag you put here

Reply:
@ChidiPatrick  ii have done the changes